services/workflows-service/docker-compose.otel.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  tracing:
+    image: jaegertracing/all-in-one:latest
```

Comment:
Can you use a fixed version instead of `latest`? It will be a bit more stable, and identical to our prod env.

Reply:
Its only for dev purposes, on production we're using Tempo and Grafana not jaeger.

Reply:
I agree that it would be better to persist an explicit version, Done